actors/telemetry-core/src/main/java/org/sunbird/util/lmaxdisruptor/SunbirdTelemetryEventConsumer.java

```diff
@@ -0,0 +1,104 @@
+package org.sunbird.util.lmaxdisruptor;
+
```

Comment:
Why two telemetry pom projects are required in lmw-mw? Remove the one unnecessary.

Reply:
i can't see two branch.